migrations/frontend/1685695443_codeowners_stats_and_counts/up.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS codeowners_owners (
+    id SERIAL NOT NULL PRIMARY KEY,
+    -- We just keep the reference as opposed to splitting it to handle or email
+    -- since the distinction is not relevant for query, and this makes indexing
+    -- way easier.
+    reference TEXT NOT NULL
+);
+
+COMMENT ON TABLE codeowners_owners IS 'Text reference in CODEOWNERS entry to use in codeowners_individual_stats. Reference is either email or handle withouth @ in front.';
+
+CREATE INDEX IF NOT EXISTS codeowners_owners_reference ON codeowners_owners USING btree (reference);
+
+-- We choose to have a table for `ownership_path_stats` - more general than for CODEOWNERS,
+-- with a specific tree_codeowned_files_count CODEOWNERS column. The reason for that
+-- is that we aim at expanding path stats by including total owned files (via CODEOWNERS
+-- or assigned ownership), and perhaps files count by assigned ownership only.
+CREATE TABLE IF NOT EXISTS ownership_path_stats (
+    file_path_id INTEGER NOT NULL PRIMARY KEY REFERENCES repo_paths(id),
+    tree_codeowned_files_count INTEGER NULL,
+    last_updated_at TIMESTAMP NOT NULL
```

Comment:
maybe `last` is redundant?
```suggestion
    updated_at TIMESTAMP NOT NULL
```